<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Vadim Litvinov</author>
 <class>StartBattleDialog</class>
 <widget class="QDialog" name="StartBattleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start new battle...</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="mapLabel">
     <property name="text">
      <string>&amp;Map:</string>
     </property>
     <property name="buddy">
      <cstring>mapLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="mapLineEdit"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="mapButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="team1Label">
     <property name="text">
      <string>Team &amp;1 bot:</string>
     </property>
     <property name="buddy">
      <cstring>team1LineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="team1LineEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="team1Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="team1SpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="team2Label">
     <property name="text">
      <string>Team &amp;2 bot:</string>
     </property>
     <property name="buddy">
      <cstring>team2LineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="team2LineEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="team2Button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="team2SpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="C" country="AnyCountry"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mapLineEdit</tabstop>
  <tabstop>mapButton</tabstop>
  <tabstop>team1LineEdit</tabstop>
  <tabstop>team1Button</tabstop>
  <tabstop>team1SpinBox</tabstop>
  <tabstop>team2LineEdit</tabstop>
  <tabstop>team2Button</tabstop>
  <tabstop>team2SpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StartBattleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StartBattleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
